prov/efa/src/rxr/rxr_ep.c

```diff
@@ -811,7 +811,7 @@ static int rxr_ep_bind(struct fid *ep_fid, struct fid *bfid, uint64_t flags)
 		ret = ofi_ep_bind_av(&rxr_ep->util_ep, &av->util_av);
 		if (ret)
 			return ret;
-
+		fprintf(stderr, "mater + 3 fix\n");
```

Comment:
Please remove this.

Reply:
Sorry. Fixed in  new revision. Also, I moved atomrsp_pool_size into rxr_env, so it is together with other pool size.